### ISSUE TITLE
All Actions Failing : Repositories Not Found

### ISSUE THREAD

**Issue by FooBartn** [aquasecurity/tfsec-pr-commenter-action#60]
Action setup does not seem to be working correctly. See output below, specifically the last line.

This is happening in multiple repositories that have not had any changes to their tfsec github action, but they all use the latest @main pr commenter. 

```yaml
tfsec:
    runs-on: ubuntu-latest
    steps:
      - name: Clone repo
        uses: actions/checkout@v3

      - name: Get Rules
        uses: actions/checkout@v3
        with:
          repository: '<removed>'
          path: '.tfsec'
          token: ${{ secrets.GH_ACCESS_TOKEN }}
        
      - name: tfsec
        uses: aquasecurity/tfsec-pr-commenter-action@main
        with:
          github_token: ${{ github.token }}
```

**Output of action:**

Current runner version: '2.289.1'
Operating System
  Ubuntu
  20.04.4
  LTS
Virtual Environment
  Environment: ubuntu-20.04
  Version: 20220227.1
  Included Software: https://github.com/actions/virtual-environments/blob/ubuntu20/20220227.1/images/linux/Ubuntu2004-Readme.md
  Image Release: https://github.com/actions/virtual-environments/releases/tag/ubuntu20%2F20220227.1
Virtual Environment Provisioner
  1.0.0.0-main-20220307-1
GITHUB_TOKEN Permissions
  Actions: write
  Checks: write
  Contents: write
  Deployments: write
  Discussions: write
  Issues: write
  Metadata: read
  Packages: write
  Pages: write
  PullRequests: write
  RepositoryProjects: write
  SecurityEvents: write
  Statuses: write
Secret source: Actions
Prepare workflow directory
Prepare all required actions
Getting action download info
**Error: Unable to resolve actions. Repositories not found :**

**Comment by SebastiaanGevers** [comment]
@owenrumney yes this is still an issue. My issue is releated I think.
error: failed to create a new commenter. PR number [130] not found for "repository"

**Comment by mrogers-palmetto** [comment]
Also experiencing the same issue: `The commenter failed with the following error:
failed to create a new commenter. PR number [54] not found for palmetto/pdp-infra`

**Comment by kolomied** [comment]
@SebastiaanGevers @mariahjrogers @agnasillo-caylent Take a look at #21 please - likely the reason if you are using GitHub Enterprise. There is a PR raised with the fix (#77, pending @owenrumney's review) in case you want to give it a go.
